analog_feedback_topic: "/teensy/analog_feedback"
subgoal_topic: "/teensy/microgoal" # Send subgoal in microsecods (servo speed control)
joint_states_topic: "/servo/JointStates"
servo_goal_topic: "/servo/servo_goal"       # Publish the desired angle (in degrees) of every servo
ros_rate: 1000 # hz 
num_servos: 3